### PR TITLE
topics properly spaced for series

### LINKED ISSUE #825: series: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by deepakavs**
Fixes #825

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -393,8 +393,8 @@
       "slug": "series",
       "topics": [
         "loops",
-		    "strings",
-		    "conditionals"
+	"strings",
+	"conditionals"
```
[the remark this answers is not in the document]
Comment (deepakavs) [reply config.json]: You are right I should use branches when committing multiple PRs. I can't make up a case where we need type_conversion in finding substrings in a given string. Could you give an example? Also , I will make the changes as you mentioned and resubmit the PR.

[the remark this answers is not in the document]
Comment (FridaTveit) [reply config.json]: The input is a string but the output is a list of integers, so at some point you would have to convert from a string to an integer :) Does that make sense?

Great, thank you :) Sorry for not spotting the problem with the previous PR sooner :)

[the remark this answers is not in the document]
Comment (deepakavs) [reply config.json]: http://exercism.io/exercises/java/series/readme
To my knowledge looks like the output need not be a list of integers, instead they can be treated as strings itself as we are not performing any arithmetic/ aggregate operations on them.

[the remark this answers is not in the document]
Comment (deepakavs) [reply config.json]: I can add "type_conversion" if the above comment did not convince you :)

[the remark this answers is not in the document]
Comment (FridaTveit) [reply config.json]: The [tests](https://github.com/exercism/java/blob/master/exercises/series/src/test/java/SeriesTest.java) require the output to be a list of integers :)

[the remark this answers is not in the document]
Comment (deepakavs) [reply config.json]: Got it!

**Comment by deepakavs** [comment]
Thank you too!

**Pull request merged**